README.md

```diff
@@ -98,79 +98,50 @@ You can type ```/help``` to get details about what commands can be used, along w
 
-When you decide to publish the draft, you can request the bot to do this.
+Start drafting your proposal in the #drafting-props channel and select the appropriate tag (budget, or governance). Depending on your tag selected the bot will handle the proposal prefix. Just create a name for your proposal.
 
-If you wish to change the emojis used when drafting & reaching soft consensus prior to publishing to snapshot this can be done in consts/constants.py. Simply replace YES_VOTE / NO_VOTE or ABSTAIN_VOTE with the desired emoji id. For example: 
+Please consider and use the following stuctures for the proposals body, for uniformity and structural reasons when making a proposal draft:
 
-```
-YES_VOTE = "<:gunta:1199583728129802322>"
-```
+**Governance (improvement-props):**
 
-# Commands
+- Authors
 
-The following commands can be used to create, edit, and publish drafts.
-All commands use discords slash commands (/)
+- Abstract
 
-**Vote Draft:**
+- Definitions (If applicable)
 
-/vote_draft
+- Background
 
-**Example:**
+- Details (if applicable)
 
-```/vote_draft``` 
+- Implementation Protocol (if applicable)
 
-Upon typing the command, the bot will present you with four buttons: "Create", "Edit", "Preview", and "Delete".
+- Voting Options (required)
+Anything can be posted here, just include the words 'voting options' for now. The bot will use default settings (Adopt, Reassess, Abstain) which will be dynamcially adjustable in the future
 
-"Create" will create a new proposal.
-"Edit" will allow you to select an an existing proposal to edit.
-"Preview" will allow you to preview an existing proposal before it is published.
-"Delete" will allow you delete and existing proposal that has not been published.
+**Budget (budgetary-props)**
 
-You can use this command repeatedly to perform different actions if necessary.
+- Authors
 
-**Publish Draft:**
+- Abstract
 
-In order to publish a draft you need to use the command /publish_draft. 
+- Type
 
-Select the draft you wish to publish from the dropdown.
+- Definitions (if applicable)
 
-This will automatically publish the draft you want to the appropriate Discord channel (Gov-Budget or Governance) depending on if it is a general proposal or a budget.
+- Background
 
-If / when a published draft reaches qurom, the draft is approved and subsequently posted to Snapshot. NOTE: env vars ``ETH_ADDRESS`` and ``ETH_PRIVATE_KEY``
-need to be set.
+- Details (specifics of imbursement, etc)
 
-A Snapshot space is required to be configured and is outside the scope of this documentation. Please refer to https://docs.snapshot.org/ to configure a Snapshot space.
-
-**Example:**
-
-```/publish_draft```
-
-**Posted Response:**
-
-```
-Topic/Vote 1: new title
+- Voting Options (required)
+Anything can be posted here, just include the words 'voting options' for now. The bot will use default settings (Adopt, Reassess, Abstain) which will be dynamcially adjustable in the future
```

Comment:
Sort of confusing for this to be marked as "required" but then also explain that you just need to include a placeholder value. Can this section be left out entirely and just handled by the bot without requiring specific user input?